.gitignore

```diff
@@ -1 +1,4 @@
 logs
+.exe
+telegram-bot/.env
+telegram-bot/main
```

Comment:
why `main` here?

Reply:
Since, `.exe` was not being shown in VSCode,thus I added `main` in order to make sure it doesn't get tracked.

Reply:
Oh, I expected the binary with the name of folder when doing `go build`. But in case of VSCode it might be different.
Go ahead and add both in `.gitignore`